### DISCUSSION TITLE
Scrum Prep Milestone 3 - Second Week

### DISCUSSION THREAD

**Discussion by YanehRuiz** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#359] [Weekly Retrospective]
Weekly Development Update: Second week of Milestone 2

Hello team!

Please provide a brief comment detailing what you accomplished this week, including any challenges faced or insights gained. Make sure to submit your updates by the deadline, which is before: Friday, November 8th, 2024

Thank you for your collaboration!

As for my collaboration:

Approved and graded points and discussed the teams weekly progress

**Comment by noel-vargas** [comment]
I started brainstorming in the ltt I can make for this milestone. Also had a chat with my teammates to set up the data base and pull/push data from/to it.

**Comment by RaquelQuinones** [comment]
Started the LTT task for the first sprint of Milestone 3.

**Comment by Chosen-Juan1** [comment]
I am planning what to do for milestone 3 as well as researching potential lecture topic tasks.

**Comment by JorgeC08** [comment]
Started LTT for the first sprint

**Comment by NathBat09** [comment]
Worked and finished the #358 task

**Comment by ghost** [comment]
I havent been able to work on the project yet, but I took a task for this sprint and will make sure to complete it before then.

**Comment by javier-rivera8** [comment]
Created the tasks I will be working on this milestone.

**Comment by williammartinez10** [comment]
Finished working on #246

**Comment by Jjulianirizarry24** [comment]
Created multiple tasks for milestone 3

**Comment by johnsantiago9** [comment]
Started brainstorming and looking at the lectures topics to developed a task for this first sprint

**Comment by JuanJRivera** [comment]
I was working on the research for the completion of the log book.

**Comment by JorgeRivera94** [comment]
Attended our team meeting, where milestone tasks where discussed. Identified some tasks. Will continue to look for lecture topic tasks.

**Comment by CaribbeanCool** [comment]
Haven't done a lot. Brainstorming for tasks.

**Comment by yadrielsamir** [comment]
Working on issue #342 and some LT tasks.

**Comment by jonVargas1** [comment]
Worked on issue #297

**Comment by Rodal1** [comment]
Worked on refactoring all week.

**Comment by Alejandro-DeJesus** [comment]
Conducted Team meeting, where the milestone goals where discussed, and Identified various issues for the milestone. Looking to start progress on the issues during the weekend.

**Comment by evnd04** [comment]
Mostly reviewed issues and started to create team lead tasks.

**Comment by lex939** [comment]
Opened all documentation feedback issues.

Coming up: Opening up issues specifically for new documentation material, organizing edits to documentation, and looking into Lecture Topic tasks for documentation.

**Comment by misaelmercado1** [comment]
Started working in my final lecture topic task and researching for my last ltt
